--- fltk-1.3.3/CMake/install.cmake~	2014-09-27 08:41:06.000000000 +0800
+++ fltk-1.3.3/CMake/install.cmake	2015-08-14 16:59:34.606534063 +0800
@@ -74,7 +74,7 @@
    macro(INSTALL_MAN FILE LEVEL)
    install(FILES
       ${FLTK_SOURCE_DIR}/documentation/src/${FILE}.man
-      DESTINATION man/man${LEVEL}
+      DESTINATION share/man/man${LEVEL}
       RENAME ${FILE}.${LEVEL}
    )
    endmacro(INSTALL_MAN FILE LEVEL)
