diff -urN fltk-1.3.0/cairo/CMakeLists.txt fltk-1.3.0-patched/cairo/CMakeLists.txt
--- fltk-1.3.0/cairo/CMakeLists.txt	2010-04-06 09:33:58.000000000 +1100
+++ fltk-1.3.0-patched/cairo/CMakeLists.txt	2012-09-14 01:39:44.173275271 +1100
@@ -8,7 +8,7 @@
 set(cairo_SRCS Fl_Cairo.cxx)
 
 #######################################################################
-add_library(fltk_cairo ${cairo_SRCS})
+add_library(fltk_cairo STATIC ${cairo_SRCS})
 
 if(MSVC)
    set_target_properties(fltk_cairo
diff -urN fltk-1.3.0/fluid/CMakeLists.txt fltk-1.3.0-patched/fluid/CMakeLists.txt
--- fltk-1.3.0/fluid/CMakeLists.txt	2011-01-06 20:24:58.000000000 +1000
+++ fltk-1.3.0-patched/fluid/CMakeLists.txt	2012-09-14 01:40:08.347275014 +1100
@@ -22,7 +22,7 @@
 )
 
 add_executable(fluid ${CPPFILES})
-target_link_libraries(fluid fltk fltk_images fltk_forms)
+target_link_libraries(fluid fltk fltk_images fltk_forms dl)
 
 # link in optional libraries
 if(FLTK_HAVE_CAIRO)
diff -urN fltk-1.3.0/src/CMakeLists.txt fltk-1.3.0-patched/src/CMakeLists.txt
--- fltk-1.3.0/src/CMakeLists.txt	2011-01-12 19:24:03.000000000 +1000
+++ fltk-1.3.0-patched/src/CMakeLists.txt	2012-09-14 01:39:44.173275271 +1100
@@ -333,6 +333,8 @@
    set_target_properties(fltk_SHARED PROPERTIES OUTPUT_NAME fltk)
 endif(MSVC)
 
+target_link_libraries(fltk_SHARED dl)
+
 if(USE_THREADS)
    target_link_libraries(fltk_SHARED ${CMAKE_THREAD_LIBS_INIT})
 endif(USE_THREADS)
@@ -383,6 +385,8 @@
    set_target_properties(fltk_forms_SHARED PROPERTIES OUTPUT_NAME fltk_forms)
 endif(MSVC)
 
+target_link_libraries(fltk_forms_SHARED dl)
+
 if(USE_THREADS)
    target_link_libraries(fltk_SHARED ${CMAKE_THREAD_LIBS_INIT})
 endif(USE_THREADS)
@@ -394,7 +398,7 @@
 #######################################################################
 add_library(fltk_images_SHARED SHARED ${IMGCPPFILES})
 target_link_libraries(fltk_images_SHARED fltk
-   ${FLTK_PNG_LIBRARIES} ${FLTK_JPEG_LIBRARIES} ${FLTK_ZLIB_LIBRARIES}
+   ${FLTK_PNG_LIBRARIES} ${FLTK_JPEG_LIBRARIES} ${FLTK_ZLIB_LIBRARIES} dl
 )
 set_target_properties(fltk_images_SHARED
    PROPERTIES CLEAN_DIRECT_OUTPUT 1
@@ -424,7 +428,7 @@
 #######################################################################
 if(OPENGL_FOUND)
    add_library(fltk_gl_SHARED SHARED ${GLCPPFILES})
-   target_link_libraries(fltk_gl_SHARED fltk ${OPENGL_LIBRARIES})
+   target_link_libraries(fltk_gl_SHARED fltk ${OPENGL_LIBRARIES} dl)
    set_target_properties(fltk_gl_SHARED
       PROPERTIES CLEAN_DIRECT_OUTPUT 1
       VERSION ${FLTK_VERSION_MAJOR}.${FLTK_VERSION_MINOR}
