--- fltk-1.3.3/CMake/install.cmake~	2015-08-15 15:36:28.947402327 +0800
+++ fltk-1.3.3/CMake/install.cmake	2015-08-15 16:02:41.641006985 +0800
@@ -70,6 +70,15 @@
    DESTINATION ${FLTK_CONFIG_PATH}
 )
 
+configure_file(
+   ${FLTK_SOURCE_DIR}/fltk-config.cmake.in
+   ${FLTK_BINARY_DIR}/fltk-config
+   @ONLY
+)
+install(PROGRAMS ${FLTK_BINARY_DIR}/fltk-config
+   DESTINATION bin
+)
+
 if(UNIX)
    macro(INSTALL_MAN FILE LEVEL)
    install(FILES
--- fltk-1.3.3/fltk-config.cmake.in~	2013-04-07 00:15:17.000000000 +0800
+++ fltk-1.3.3/fltk-config.cmake.in	2015-08-15 16:07:14.808585610 +0800
@@ -31,9 +31,9 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
 exec_prefix_set=no
-bindir=@PREFIX_BIN@
-includedir=@PREFIX_INCLUDE@
-libdir=@PREFIX_LIB@
+bindir=${prefix}/bin
+includedir=%{prefix}/include
+libdir=@CMAKE_INSTALL_LIBDIR@
 srcdir=.
 
 # compiler names
