--- a/CMake/install.cmake
+++ b/CMake/install.cmake
@@ -50,7 +50,7 @@
 set(CONFIG_PATH ${CMAKE_INSTALL_PREFIX}/${FLTK_CONFIG_PATH})
 
 install(EXPORT FLTK-Targets
-   DESTINATION ${FLTK_CONFIG_PATH}
+   DESTINATION ${FLTK_CONFIG_PATH}/cmake
    FILE FLTK-Targets.cmake
 )
 
@@ -63,7 +63,7 @@
 install(FILES
     ${FLTK_BINARY_DIR}/etc/FLTKConfig.cmake
     ${FLTK_SOURCE_DIR}/CMake/FLTK-Functions.cmake
-    DESTINATION ${FLTK_CONFIG_PATH}
+    DESTINATION ${FLTK_CONFIG_PATH}/cmake
 )
 
 configure_file(
@@ -73,7 +73,7 @@
 )
 
 install(FILES ${FLTK_BINARY_DIR}/etc/UseFLTK.cmake
-   DESTINATION ${FLTK_CONFIG_PATH}
+   DESTINATION ${FLTK_CONFIG_PATH}/cmake
 )
 
 # generate fltk-config
